exec/cnex/dictionary.h

```diff
@@ -18,12 +18,16 @@ typedef struct tagTDictionary {
 } Dictionary;
 
 Dictionary *dictionary_createDictionary(int64_t size);
-
 void dictionary_freeDictionary(Dictionary *self);
 
-struct tagTCell *dictionary_findDictionaryEntry(Dictionary *self, struct tagTString *key, struct tagTCell *def);
+Dictionary *dictionary_copyDictionary(Dictionary *self);
+int dictionary_compareDictionary(Dictionary *lhs, Dictionary *rhs);
+
+struct tagTCell *dictionary_findDictionaryEntry(Dictionary *self, struct tagTString *key);
 void dictionary_addDictionaryEntry(Dictionary *self, struct tagTString *key, struct tagTCell *value);
 
+int64_t dictionary_findIndex(Dictionary *self, struct tagTString *key);
```

Comment:
This function is internal and shouldn't be in the header file. Make it `static` in `dictionary.c`.

Reply:
Opps.  Fixed.